src/config.rs

```diff
@@ -22,6 +29,168 @@ impl fmt::Display for InstallError {
+
+// XXX now that we support text effects, "color scheme" doesn't seem accurate anymore. Let me know if you'd prefer a different name. The good thing about `Styles` is, that it's pretty short. "Theme" would be another good name, I just realized.
+
+/// A struct that represents styles that should be used by `color_eyre`
+#[derive(Debug, Copy, Clone, Default)]
+pub struct Styles {
+    pub(crate) file: Style,
+    pub(crate) line_number: Style,
+    pub(crate) spantrace_target: Style,
+    pub(crate) spantrace_fields: Style,
+    pub(crate) active_line: Style,
+    pub(crate) error: Style,
+    pub(crate) help_info_note: Style,
+    pub(crate) help_info_warning: Style,
+    pub(crate) help_info_suggestion: Style,
+    pub(crate) help_info_error: Style,
+    pub(crate) dependency_code: Style,
+    pub(crate) crate_code: Style,
+    pub(crate) code_hash: Style,
+    pub(crate) panic_header: Style,
+    pub(crate) panic_message: Style,
+    pub(crate) panic_file: Style,
+    pub(crate) panic_line_number: Style,
+    pub(crate) hidden_frames: Style,
+}
+
+macro_rules! style_setters {
+    ($(#[$meta:meta] $name:ident),* $(,)?) => {
+        $(
+            #[$meta]
+            pub fn $name(mut self, style: Style) -> Self {
+                self.$name = style;
+                self
+            }
+        )*
+    };
+}
+
+impl Styles {
+    /// Blank styles
+    pub fn new() -> Self {
+        Self::default()
+    }
+
+    /// Styles for dark background. This is the default
+    pub fn dark() -> Self {
+        Self {
+            file: style().purple(),
+            line_number: style().purple(),
+            active_line: style().white().bold(),
+            error: style().bright_red(),
+            help_info_note: style().bright_cyan(),
+            help_info_warning: style().bright_yellow(),
+            help_info_suggestion: style().bright_cyan(),
+            help_info_error: style().bright_red(),
+            dependency_code: style().green(),
+            crate_code: style().bright_red(),
+            code_hash: style().bright_black(),
+            panic_header: style().red(),
+            panic_message: style().cyan(),
+            panic_file: style().purple(),
+            panic_line_number: style().purple(),
+            hidden_frames: style().bright_cyan(),
+
+            // XXX The following two are specific to `color_spantrace`. See my comment there for my recommendation.
+            spantrace_target: style().color(XtermColors::UserBrightRed),
+            spantrace_fields: style().color(XtermColors::UserBrightCyan),
+        }
+    }
+
+    // XXX it would be great, if someone with more style optimizes the light theme below. I just fixed the biggest problems, but ideally there would be darker colors (however, the standard ANSI colors don't seem to have many dark enough colors. Maybe xterm colors or RGB colors would be better (however, again, see my comment regarding xterm colors in `color_spantrace`))
```

Comment:
I'll try testing it out locally later today, but I'm also not really an expert on these things, all the coloring in this crate is based on the theme from `color-backtrace` :D

Reply:
👍

I'll keep the "XXX" comments then.